corehq/apps/users/models.py

```diff
@@ -1414,7 +1416,7 @@ def clear_quickcache_for_user(self):
     def save(self, **params):
         super(CommCareUser, self).save(**params)
 
-        from corehq.apps.users.signals import commcare_user_post_save
+        from .signals import commcare_user_post_save
```

Comment:
No particular reason - I was adding the same import before each of the signal connections, so I just made them consistent, and I generally prefer explicit relative imports for stuff within the same module.  Actually, I could probably just import this at the top of the file now that the signal binding has moved to the AppConfig.